src/PathCompletionItem.ts

```diff
@@ -34,11 +34,11 @@ export class PathCompletionItem extends CompletionItem {
         const fragments = fileInfo.file.split('.');
         const extension = fragments[fragments.length - 1];
 
-        if (extension !== documentExtension) {
+        if (extension !== documentExtension && !isImport) {
```

Comment:
This makes the expression always false since we break the function a few lines further up if it is not an import